scripts/zones/South_Gustaberg/mobs/Leaping_Lizzy.lua

```diff
@@ -4,10 +4,26 @@
 -----------------------------------
 require("scripts/globals/hunts")
 -----------------------------------
+local ID = require("scripts/zones/South_Gustaberg/IDs")
 local entity = {}
 
+entity.onMobSpawn = function(mob)
+    -- Disallow two Leaping Lizzies from spawning at the same time
```

Comment:
I thought the system already prevented that when KJ wrote it. did we break that somewhere along the way and it didn't get reported? Stray Mary will have same issue if so.

Reply:
We had this issue on Wings, although I'm not sure if changes were done on LSB to not need this. If you look at my Zulkheim PR I am also doing this for Stray Mary. More than happy to delete this if its not needed, I will give it a test after work today.

Reply:
Looks like its not a concern on here. I will go ahead and remove that code for LL and Stray Mary.